### PR TITLE
feat(vps): add tracking on upgrade and upscale

### PR DESCRIPTION
**Pull request by cbourgois**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-6144
| License          | BSD 3-Clause

## Description

Add tracking on VPS upgrade and upscale

### :flags: Translations

- [x] TRANS-32454

### CODE REVIEW

**Review by frenautvh — DISMISSED** [review]

File: packages/manager/modules/vps/src/dashboard/tile/configuration/index.html [hunk 1]
```diff
@@ -15,12 +15,19 @@
             <span data-ng-bind=":: $ctrl.vps.vcore"></span>
             <p
```
Comment (marie-j) [inline packages/manager/modules/vps/src/dashboard/tile/configuration/index.html:16]: Any reason to keep the `p` tag ?

File: packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js [hunk 2]
```diff
@@ -113,10 +113,20 @@ export default /* @ngInject */ ($stateProvider) => {
...
-      goToUpgrade: /* @ngInject */ ($state) => (upgradeType) =>
-        $state.go(`vps.detail.dashboard.configuration.upgrade`, {
+      goToUpgrade: /* @ngInject */ ($state, stateVps) => (upgradeType) => {
+        let from = 0;
+        if (upgradeType === 'memory') {
+          from = stateVps.model[upgradeType] / 1024;
+        } else if (upgradeType === 'storage') {
+          from = stateVps.model.disk;
+        }
+        const to = from * 2;
```
Comment (marie-j) [inline packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js:123]: Always the case ?

Comment (cbourgois) [reply packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js:123]: Seems to be : https://github.com/ovh/manager/blob/develop/packages/manager/modules/vps/src/dashboard/tile/configuration/upgrade/routing.js#L132-L141

File: packages/manager/modules/vps/src/dashboard/tile/configuration/index.html (hunk 1 above)
Comment (cbourgois) [reply packages/manager/modules/vps/src/dashboard/tile/configuration/index.html:16]: Yes, `span + p > a`, to avoid `a.d-block`

**Review by antleblanc — APPROVED** [review]

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**